src/Modules/OrchardCore.Commerce.Payment/Manifest.cs

```diff
@@ -0,0 +1,29 @@
+using OrchardCore.Modules.Manifest;
+using static OrchardCore.Commerce.ContentFields.Constants.FeatureIds;
+using static OrchardCore.Commerce.Payment.Constants.FeatureIds;
+using static OrchardCore.Commerce.Tax.Constants.FeatureIds;
+
+[assembly: Module(
+    Name = "Orchard Core Commerce - Payment",
+    Author = "The Orchard Team",
+    Website = "https://github.com/OrchardCMS/OrchardCore.Commerce",
+    Version = "0.0.1",
+    Description = "Payment for Orchard Core Commerce.",
+    Category = "Commerce"
+)]
+
+[assembly: Feature(
+    Id = Payment,
+    Name = "Orchard Core Commerce - Payment",
+    Category = "Commerce",
+    Description = "Payment for Orchard Core Commerce.",
+    Dependencies = new[] { ContentFields, Tax }
+)]
+
+[assembly: Feature(
+    Id = DummyProvider,
+    Name = "Orchard Core Commerce - Payment - Dummy Provider",
+    Category = "Commerce",
+    Description = "Dummy payment provider used for development and testing.",
```

Comment:
Not necessary, because the feature is [completely bypassed in production](https://github.com/OrchardCMS/OrchardCore.Commerce/blob/637f427f1b441381ffc620bf21503910ecd67c19/src/Modules/OrchardCore.Commerce.Payment/Services/DummyPaymentProvider.cs#L37-L41) anyway.

Reply:
Ah I see, okay.